src/locale/defaults.h

```diff
@@ -66,6 +66,9 @@
 #ifndef D_STR_ECONO
 #define D_STR_ECONO "Econo"
 #endif  // D_STR_ECONO
+#ifndef D_STR_ECOCOOL
+#define D_STR_ECOCOOL "Ecocool"
```

Comment:
[Q] What does this feature do? If we can, we want to reuse an existing term/word/name etc to reduce translation effort and to reduce the space required for the strings.
e.g. Would `Econo` be enough?

Reply:
I used Mitsubishis terminology from the remote and from the user’s manual. In this way it’s consistent for the next (Mitsubishi) user in line. I guess it’s a trademarked term, and to mee, it would be confusing to use something else that what the manufacturer uses. That would mean, I would have to look into the code to check the functionality.
The functionality is that it uses a (infrared, I guess) sensor to “see” if there are persons in the room. In combination with other settings, that can be used to adjust heat depending on whether there are persons present. It can also be used for providing direct or indirect airflow towards the persons in the room.